### ISSUE TITLE
Option to dismiss Default Launcher "Welcome back!" message

### ISSUE THREAD

**Issue by 2mcdev** [8bitPit/Niagara-Issues#1541]
TL;DR: Make it possible to suppress the "Welcome back!" message for people who use apps like Tasker to conditionally switch between launchers.

-- 
On my Galaxy G Fold 2 I'd like to use Niagara when my phone is in folded mode and Nova when I use my phone in tablet mode. For this reason I have set Tasker AutoTools as my Default home app, which automaticly starts the right launcher depending on my phone being folded or opened to tablet size.

This works perfectly. However I cannot seem to get rid of "Welcome back! Do you want to set Niagara Launcher as default again?" message. It would be great if you could make it possible to permanently dismiss this message.

**Comment by 8bitPit** [comment]
There you go :) https://help.niagaralauncher.app/article/100-foldable-device-support

**Comment by 2mcdev** [comment]
Wow i really need to reup my search skills. Thanks!

**Comment by Amorphous404** [comment]
- Duplicate of #949
